packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -203,19 +203,22 @@ export class GCFBootstrapper {
         auth: await this.getInstallationToken(body.installation.id),
       });
       // Installations API documented here: https://developer.github.com/v3/apps/installations/
-      const {data} = await octokit.request('/installation/repositories', {
+      for await (const response of octokit.paginate.iterator({
```

Comment:
This is kinda hard to read.  Maybe perform the call to `octokit.paginate.iterator`, and then `for await` over that result instead of inlining it?